### ISSUE TITLE
What are the polynomial points for F(4,5)???

### ISSUE THREAD

**Issue by victorygogogo** [andravin/wincnn#22]

**Comment by andravin** [comment]
Because F(4,5) is the same degree as F(6,3), the same 7 interpolation points should work for both algorithms.

Generally for any F(m,r) with m+r-2 = n, you can try the same n interpolation points.

Following the example in the README, try
```
>>> from sympy import Rational
>>> wincnn.showCookToomFilter((0,1,-1,2,-2,Rational(1,2),-Rational(1,2)), 4, 5)
```

Hope that helps.

**Comment by victorygogogo** [comment]
thank you very much

**Comment by victorygogogo** [comment]
(0,1,-1,Rational(1,2),-Rational(1,2),2,-2  this interp can get the right result ?

**Comment by andravin** [comment]
Yes, the order you list the points in does not matter .. and otherwise your list is the same as mine.

The usual caveat applies: the more interpolation points your algorithm uses, the greater the numeric error when computed with limited precision arithmetic.

Increasingly, I am only interested in the shortest convolution algorithms, because the latest hardware tends not to have enough memory bandwidth to make the larger algorithms efficient. Also F(2,3) can be accurate even with low precision arithmetic.

**Comment by victorygogogo** [comment]
so if use low precision data type  such as float16  ,f(2,3) can ignore the error ?

**Comment by andravin** [comment]
Yes, F(2,3) should work well with float16 precision, but as always, actually measure the error compared to a high precision direct convolution for your application!

**Comment by Shipley1105** [comment]
How to compute F(7,3)?
